FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FirebaseRemoteConfig/Sources/RCNConfigRealtime.h"
+#import <Foundation/Foundation.h>
+#import "FirebaseRemoteConfig/Sources/RCNConfigConstants.h"
+
+NSInteger listenerCount = 1;
+
+@implementation FIRConfigUpdateListenerRegistration {
+  RCNConfigRealtime *_realtimeClient;
+  NSString *listenerName;
```

Comment:
ivar should be prefixed with "_" so this one is _listenerName

Reply:
Updated